consensus/bor/heimdall/client.go

```diff
@@ -208,7 +276,7 @@ retryLoop:
 
 				continue retryLoop
 			}
-
+			log.Error("Result of fetching", "Result", result)
```

Comment:
Is it an error?

Reply:
Sorry , I used it for testing, Forgot to remove it